Ginger/GingerCore/Environments/Database.cs

```diff
@@ -272,7 +272,7 @@ public string GetConnectionString()
                 connStr = "Data Source=" + TNSCalculated + ";User Id=" + UserCalculated + ";";
 
                 String deCryptValue = EncryptionHandler.DecryptString(PassCalculated, ref res, false);
-                ConnectionString = connStr + "Password=" + Pass + ";";
+                //ConnectionString = connStr + "Password=" + Pass + ";";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5100039)